### DISCUSSION TITLE
Display sleep timer?

### DISCUSSION THREAD

**Discussion by FinlessBob** [Jyers/Marlin#1086] [Ideas]
Is it possible to add a display off sleep timer that starts when printing starts?  After set time the display turns off.  When the print is done it turns back on.

**Comment by Reggi3** [comment]
That sounds like a good idea, perhaps even a screensaver style timeout on it for the whole system might be useful?  I know there is the option to turn the display off in the Control>Visual menu, that might help you for now?

**Reply by FinlessBob** [reply]
Yes I have been going into tune menu and turning the screen off while printing.  Great feature!
